### ISSUE TITLE
Document required directory structure

### ISSUE THREAD

**Issue by thibaudcolas** [torchbox/django-pattern-library#21]
In GitLab by @bcdickinson on Oct 23, 2018, 09:28

Mikalab is very fussy about how the pattern templates of laid out in the file system. From CCA:

```
.
└── patterns
    ├── atoms
    │   ├── breadcrumb
    │   ├── crosshair
    │   ├── image
    │   ├── select
    │   ├── story-meta
    │   └── video-embed
    ├── molecules
    │   ├── breadcrumb-block
    │   ├── course_block
    │   ├── cta-large
    │   ├── cta-small
    │   ├── division-teaser
    │   ├── embed-with-text
    │   ├── feature
    │   ├── gallery-item
    │   ├── image-with-text
    │   ├── info-header
    │   ├── keydate
    │   ├── listing-item
    │   ├── media-panel
    │   ├── multicolumn-list
    │   ├── navigation
    │   ├── nav-links
    │   ├── pagination
    │   ├── person-snippet
    │   ├── program-card
    │   ├── program-hero
    │   ├── quote
    │   ├── related-topic
    │   ├── search-field
    │   ├── search-form
    │   ├── search-results
    │   ├── section-heading
    │   ├── section-teaser
    │   ├── section_tile
    │   ├── streamfield
    │   │   └── blocks
    │   ├── student-work-gallery-item
    │   ├── teaser-block
    │   └── video-hero
    ├── organisms
    │   ├── divisions
    │   ├── footer
    │   ├── hero-index
    │   ├── listing-items
    │   ├── main
    │   ├── newsroom-hero
    │   ├── programs-grid
    │   ├── related-programs
    │   ├── related-topics
    │   ├── search-panel
    │   ├── search-result
    │   ├── section_grid
    │   ├── streamfield
    │   └── student-work-gallery
    ├── pages
    │   ├── academics
    │   ├── home
    │   ├── index_page
    │   ├── info
    │   ├── newsroom
    │   ├── program
    │   ├── search
    │   └── story
    ├── _pattern_library_only
    │   └── streamfield
    └── templates
        ├── academics
        ├── home
        ├── index_page
        ├── info
        ├── newsroom
        ├── program
        ├── search
        └── story
```

Each subdirectory contains name-matched pairs of `<blah>.html` and `<blah>.yaml` files.

**Comment by thibaudcolas** [comment]
In GitLab by @m1kola on Oct 23, 2018, 09:30

Thanks. For whoever will be working on this: please, remove [this todo](https://git.torchbox.com/tech/django-pattern-library/blob/85e28899376d47e8a2151ef378458661dc53fe7f/docs/README.md#L322) from the docs.

**Comment by thibaudcolas** [comment]
In GitLab by @alxbridge on Oct 23, 2018, 11:56

created branch [`18-document-required-directory-structure`](https://git.torchbox.com/tech/django-pattern-library/compare/master...18-document-required-directory-structure)

**Comment by thibaudcolas** [comment]
In GitLab by @alxbridge on Oct 24, 2018, 10:29

closed via commit 65ec78b520dde8998e78d003583628028996b642

**Comment by thibaudcolas** [comment]
In GitLab by @alxbridge on Oct 24, 2018, 10:29

closed via merge request !15
